File: Javascript/3-Basic-Actions/5-Get-Text.js
const { Builder, By } = require('selenium-webdriver');

(async function(){
    let driver = await new Builder().forBrowser('chrome').build();
    try{
        await driver.get('https://www.w3schools.com/default.asp');
        const my_left_nav = await driver.findElement(By.className('w3-bar-block'));
        const left_nav_text = await my_left_nav.getText();
        console.log(left_nav_text);
    } finally {
        driver.quit()
    }
})()

File: Python/3-Basic-Actions/5-Get-Text.py
from selenium import webdriver
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver = webdriver.Chrome()

try:
    driver.get('https://www.w3schools.com/default.asp')
    my_left_nav = driver.find_element_by_class_name('w3-bar-block')
    left_nav_text = my_left_nav.text

    print(left_nav_text)

finally:
    driver.quit()
